src/test/java/seedu/address/logic/commands/MarkGroupAttendanceCommandTest.java

```diff
@@ -0,0 +1,99 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.person.Group;
+import seedu.address.model.person.Person;
+import seedu.address.testutil.PersonBuilder;
+
+public class MarkGroupAttendanceCommandTest {
```

Comment:
Nice test cases! Can consider including a test cases for duplicate markings too?

Reply:
This is only if we keep the functionality where we prevent the users from duplicate markings